core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3144,15 +3144,29 @@ public void activeSyncMetadata(AlluxioURI path, Collection<AlluxioURI> changedFi
 
   @Override
   public boolean recordActiveSyncTxid(long txId, long mountId) {
-    try (RpcContext rpcContext = createRpcContext()) {
+    MountInfo mountInfo = mMountTable.getMountInfo(mountId);
+    AlluxioURI mountPath = null;
+    if (mountInfo != null) {
+      mountPath = mountInfo.getAlluxioUri();
+    } else {
+      return false;
+    }
 
+    LockingScheme lockingScheme = new LockingScheme(mountPath, LockPattern.READ, false);
```

Comment:
no need for lockingScheme if we aren't performing ufs sync. Can just inline the path and pattern into the call to `lockInodePth`

Reply:
fixed.